test/test_transforms_v2.py

```diff
@@ -551,7 +551,7 @@ def affine_bounding_boxes(bounding_boxes):
 
 class TestResize:
     INPUT_SIZE = (17, 11)
-    OUTPUT_SIZES = [17, [17], (17,), None, [12, 13], (12, 13)]
+    OUTPUT_SIZES = [17, [17], (17,), [12, 13], (12, 13), None]
```

Comment:
I put the value of None in the middle of the list originally because some of the tests (for example `test_kernel_mask` and `test_kernel_video`) use `OUTPUT_SIZES[-1]` as the value for size, and I felt that those tests weren't intended to be run using `size = None`. 

Some of the tests also use `OUTPUT_SIZES[0]` (e.g. test_pil_interpolation_compat_smoke), so I felt that putting None at the beginning of the list would be improper as well.

Reply:
Ah good point, I missed that when I did the changes, thanks!